hardware/tftp/Android.mk

```diff
@@ -18,25 +18,41 @@ target_prefixes := \
     slpi \
     cdsp
 
+target_soc := \
+    msm
+
+ifeq ($(TARGET_USES_ESOC),true)
+target_soc += \
+    mdm
+endif
```

Comment:
You know by now I'm a man of useless and unreasonable requests:

```suggestion
target_soc := \
    msm \
    $(if $(filter true,$(TARGET_USES_ESOC)),mdm,)
```

Reply:
That's ugly and it's a big no for me.

Reply:
That's what I said :) My itch prefers conciseness over ugliness :D